engine/src/main/java/org/terasology/logic/console/Console.java

```diff
@@ -25,6 +25,9 @@
 /**
  */
 public interface Console {
+
+    String NEW_LINE = "\n";
```

Comment:
I guess this could be moved to an even more general interface - maybe `TerasologyContants`? There are many other places where `\n` was hardcoded and could be replaced by a proper constant.